operatorhub/openshift/config.yaml

```
@@ -246,6 +246,13 @@ image-substitutions:
         envKeys:
           - IMAGE_MAG_TEKTON_TASKGROUP_CONTROLLER
           - IMAGE_MAG_MANUAL_APPROVAL
+- image: registry.redhat.io/openshift-pipelines/pipelines-tektoncd-pruner-controller-rhel9@
```

How about this name
```suggestion
- image: registry.redhat.io/openshift-pipelines/pipelines-pruner-controller-rhel9@
```
I think @vbobade@redhat.com created name with above

---

Done